app/Member.php

```diff
@@ -9,10 +9,13 @@
 class Member extends Model
 {
     protected $table="members";
```

Comment:
This is a code style error, there should be spaces around `=`

Reply:
yep sr